src/style.css

```diff
@@ -0,0 +1,148 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 198px;
+  background-color: #fff;
+  box-sizing: content-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__conteiner {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 159px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.img__imac {
```

Comment:
`img` block should be in separate file. But for this task you shoud remove it at all. Check the checklist and the task.)